docs/settings/apm-settings.asciidoc

```diff
@@ -13,3 +13,16 @@ You do not need to configure any settings to use APM. It is enabled by default.
 
 xpack.apm.enabled:: Set to `false` to disabled the APM plugin {kib}. Defaults to
 `true`.
+
+xpack.apm.ui.enabled:: Set to `false` to hide the APM plugin {kib} from the menu. Defaults to
+`true`.
+
+apm_oss.indexPattern:: Index pattern used to query APM data. Defaults to `apm-&#42;`.
```

Comment:
Can you be a bit more specific what this is used for, as below the index patterns for the specific events are listed.

Reply:
Something like: _Index pattern is for integrations with Machine Learning and Kuery Bar. It must match all apm indices._?